src/main/webapp/register/index.jsp

```diff
@@ -0,0 +1,42 @@
+<!--
+  ~ Copyright 2020 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Registration</title>
+  <%@ include file="/bootstrap-css.html" %>
+  <script src="maps.js"></script>
+  <script defer
+      src="https://maps.googleapis.com/maps/api/js?key=<API_KEY_HERE>&callback=loadMap&libraries=places">
```

Comment:
I've updated this now to rely on an environment variable. For development, the development key can be set via the terminal, and for deployment, the production key will have to be set as an environment variable in the appengine-web.xml configuration file.

Since App Engine is a serverless environment, I couldn't rely on file I/O, so I thought this would be a workable idea.

Please do let me know what you think about the approach, and if there's anything you think I could improve upon.